src/utils/api.js

```diff
@@ -0,0 +1,10 @@
+import { CATEGORIES, WATCHLIST } from 'constants/constants';
+
+export const setCategories = category => localStorage.setItem(CATEGORIES, category);
+
+export const getCategories = () => localStorage.getItem(CATEGORIES);
+
+export const setWatchlist = array => {
+  return localStorage.setItem(WATCHLIST, JSON.stringify(array));
+};
+export const getWatchlist = () => localStorage.getItem(WATCHLIST);
```

Comment:
What do you think about returning an empty array here if this is `null` or `undefined`? This way, we don't need to call `setWatchlist` with an empty array the first time
```suggestion
export const getWatchlist = () => localStorage.getItem(WATCHLIST) || [];
```